### DISCUSSION TITLE
Welcome to Simple-DankGrinder Discussions!

### DISCUSSION THREAD

**Discussion by xHaruke** [xHaruke/Simple-DankGrinder#1] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RAINBOWgamesYT** [comment]
hi im RAINBOWgames (i don't feel like saying my real name :/) so i wanted to ask a question about the dank memer bot well its more of a suggestion but instead of the start and end time would it be possible for you to make it to where instead of the start and end time it can have something like a timer where you put a specific time it runs for( just make the limit 12 hours though) and have a time that you have to wait before you can run it again or have it a automatic thing where after the time set by the person it has to wait a bit before it can do the process again. please if you can make a version of it like this it would make it so much easier and less confusing to use. 
            thank you for your time  :D !!

**Comment by xHaruke** [comment]
@RAINBOWgamesYT I don't get why you would want to remove a feature but adding that won't be too hard I'll just have to remove the startTime and endTime and add fixed value there instead of the user controlling it.. but I really don't get it why you would need that I mean I have provided with examples too you can just enter that value

**Reply by RAINBOWgamesYT** [reply]
oh well sorry if i gave u any trouble i'm kinda new to this stuff

**Reply by xHaruke** [reply]
its alright

**Comment by RAINBOWgamesYT** [comment]
npm run start

> simple-dankgrinder@1.0.0 start
> node index.js

Start Time : 12:XX PM
End Time : 12:XX PM
Runtime :  12 hrs

Auto Buy Running !

Logged in as RAINBOWgames#4150

      [OK] Discord.js-selfbot-v13 is up to date. Current: 2.4.7
  
      If you don't want to show this message, set `checkUpdate` to false
  
      new Client({
          checkUpdate: false,
      });
  
      and using event update
      https://discordjs-self-v13.netlify.app/#/docs/docs/main/class/Client?scrollTo=e-update
> Resting...
01/08 # 11:53 AM
> Resting...
01/08 # 11:53 AM
> Resting...
01/08 # 11:53 AM
> Resting...
01/08 # 11:53 AM
> Resting...
01/08 # 11:53 AM
> Resting...
01/08 # 11:54 AM
> Resting...
01/08 # 11:54 AM
> Resting...
01/08 # 11:54 AM
> Resting...
01/08 # 11:54 AM
> Resting...
01/08 # 11:54 AM
> Resting...
01/08 # 11:54 AM
> Resting...
01/08 # 11:54 AM
> Resting...
01/08 # 11:54 AM
> Resting...
01/08 # 11:54 AM
> Resting...
01/08 # 11:55 AM
> Resting...
01/08 # 11:55 AM
> Resting...
01/08 # 11:55 AM
> Resting...
01/08 # 11:55 AM
> Resting...
01/08 # 11:55 AM
> Resting...
01/08 # 11:56 AM
> Resting...
01/08 # 11:56 AM
> Resting...
01/08 # 11:56 AM
> Resting...
01/08 # 11:56 AM
> Resting...
01/08 # 11:56 AM
> Resting...
01/08 # 11:56 AM
> Resting...
01/08 # 11:56 AM
> Resting...
01/08 # 11:56 AM
> Resting...
01/08 # 11:57 AM
> Resting...
01/08 # 11:57 AM
> Resting...
01/08 # 11:57 AM
> Resting...
01/08 # 11:57 AM
> Resting...
01/08 # 11:57 AM
> Resting...
01/08 # 11:58 AM
> Resting...
01/08 # 11:58 AM
> Resting...
01/08 # 11:58 AM
> Resting...
01/08 # 11:58 AM
> Resting...
01/08 # 11:58 AM
> Resting...
01/08 # 11:58 AM
> Resting...
01/08 # 11:58 AM
> Resting...
01/08 # 11:58 AM
> Resting...
01/08 # 11:59 AM
> Resting...
01/08 # 11:59 AM
> Resting...
01/08 # 11:59 AM
> Resting...
01/08 # 11:59 AM
> Resting...
01/08 # 11:59 AM
> Resting...
01/08 # 11:59 AM
> Resting...
01/08 # 12:00 PM
> Resting...
01/08 # 12:00 PM
> Resting...
01/08 # 12:00 PM
> Resting...
01/08 # 12:00 PM
> Resting...
01/08 # 12:00 PM
> Resting...
01/08 # 12:00 PM
> Resting...
01/08 # 12:01 PM
> Resting...
01/08 # 12:01 PM
> Resting...
01/08 # 12:01 PM
> Resting...
01/08 # 12:01 PM
> Resting...
01/08 # 12:01 PM
> Resting...
01/08 # 12:01 PM
> Resting...
01/08 # 12:01 PM
> Resting...
01/08 # 12:01 PM
> Resting...
01/08 # 12:02 PM
> Resting...
01/08 # 12:02 PM
> Resting...
01/08 # 12:02 PM
> Resting...
01/08 # 12:02 PM
> Resting...
01/08 # 12:02 PM
> Resting...
01/08 # 12:02 PM
> Resting...
01/08 # 12:03 PM
> Resting...
01/08 # 12:03 PM
> Resting...
01/08 # 12:03 PM
> Resting...
01/08 # 12:03 PM
> Resting...
01/08 # 12:03 PM
> Resting...
01/08 # 12:03 PM
> Resting...
01/08 # 12:04 PM
> Resting...
01/08 # 12:04 PM
> Resting...
01/08 # 12:04 PM
> Resting...
01/08 # 12:04 PM
> Resting...
01/08 # 12:04 PM
> Resting...
01/08 # 12:04 PM
> Resting...
01/08 # 12:04 PM
@xHaruke do you know why this is happening and how to fix it because i followed the instructions exactly as it said

**Reply by xHaruke** [reply]
@RAINBOWgamesYT you have the same start time and end time

**Reply by xHaruke** [reply]
try starttime as 11 and end time as 23

**Reply by xHaruke** [reply]
oh wait nvm the start time says 12:XX PM XX means it will run in any minute before 13 like it can run in 12:45 too

**Reply by RAINBOWgamesYT** [reply]
oh ok so there isn't anything wrong tho what do i do if it doesn't start before then

**Reply by xHaruke** [reply]
fork the new version it tells the exact time when the bot will start and stop

**Reply by RAINBOWgamesYT** [reply]
i did for it i even re did the replit bot thing but how do we put in the minutes

**Reply by RAINBOWgamesYT** [reply]
o nvm

**Reply by RAINBOWgamesYT** [reply]
is it supposed to do this 
Start Time : 11:00 AM
End Time : 11:00 PM
Runtime :  12 hrs

Auto Buy Running !

Logged in as RAINBOWgames#4150

      [OK] Discord.js-selfbot-v13 is up to date. Current: 2.4.7
  
      If you don't want to show this message, set `checkUpdate` to false
  
      new Client({
          checkUpdate: false,
      });
  
      and using event update
      https://discordjs-self-v13.netlify.app/#/docs/docs/main/class/Client?scrollTo=e-update
> Resting...
02/08 # 10:53 AM
> Resting...
02/08 # 10:53 AM
> Resting...
02/08 # 10:53 AM
> Resting...
02/08 # 10:53 AM
> Resting...
02/08 # 10:54 AM
> Resting...
02/08 # 10:54 AM
> Resting...
02/08 # 10:54 AM
> Resting...
02/08 # 10:54 AM
> Resting...
02/08 # 10:54 AM
> Resting...
02/08 # 10:55 AM
> Resting...
02/08 # 10:55 AM
> Resting...
02/08 # 10:55 AM
> Resting...
02/08 # 10:55 AM
> Resting...
02/08 # 10:55 AM
> Resting...
02/08 # 10:55 AM
> Resting...
02/08 # 10:55 AM
> Resting...
02/08 # 10:55 AM
> Resting...
02/08 # 10:56 AM
> Resting...
02/08 # 10:56 AM
> Resting...
02/08 # 10:56 AM
> Resting...
02/08 # 10:56 AM
> Resting...
02/08 # 10:56 AM
> Resting...
02/08 # 10:56 AM
> Resting...
02/08 # 10:57 AM
> Resting...
02/08 # 10:57 AM
> Resting...
02/08 # 10:57 AM
> Resting...
02/08 # 10:57 AM
> Resting...
02/08 # 10:57 AM
> Resting...
02/08 # 10:57 AM
> Resting...
02/08 # 10:58 AM
> Resting...
02/08 # 10:58 AM
> Resting...
02/08 # 10:58 AM
> Resting...
02/08 # 10:58 AM
> Resting...
02/08 # 10:58 AM
> Resting...
02/08 # 10:58 AM
> Resting...
02/08 # 10:58 AM
> Resting...
02/08 # 10:59 AM
> Resting...
02/08 # 10:59 AM
> Resting...
02/08 # 10:59 AM
> Resting...
02/08 # 10:59 AM
> Resting...
02/08 # 10:59 AM
> Resting...
02/08 # 10:59 AM
> Resting...
02/08 # 11:00 AM
> Resting...
02/08 # 11:00 AM
> Resting...
02/08 # 11:00 AM
> Resting...
02/08 # 11:00 AM
> Resting...
02/08 # 11:00 AM
> Resting...
02/08 # 11:00 AM
> Resting...
02/08 # 11:00 AM
> Resting...
02/08 # 11:00 AM
> Resting...
02/08 # 11:01 AM
> Resting...

**Reply by xHaruke** [reply]
@RAINBOWgamesYT hmmm what timezone did you put?
